app/src/main/java/com/partymusicq/android/pojo/Song.kt

```diff
@@ -0,0 +1,16 @@
+package com.partymusicq.android.pojo
+
+import com.google.firebase.firestore.IgnoreExtraProperties
+
+@IgnoreExtraProperties
+data class Song(val uri: String = "",
```

Comment:
Is this all of the data that we are going to store for a song?

Reply:
i can't think of anything else atm. anything else can be gotten with the api once you have the uri. the only data that needs to be synced between users is the position. if we're doing the voting system then that is another thing that would be added to the object